README.md

```diff
@@ -44,14 +45,22 @@ vagrant ssh
 That's it!
 
 
-### Docker
+### Docker Setup
 We provide experimental support to run the development setup in a docker container using
+Please install Docker Desktop, **not** just the CLI docker engine. Vagrant does conflict with used ports from other docker containers, even if they are not in use. If you are encountering this issue during the build process you can change the `host` port in `Vagrantfile`.
```

Comment:
> Please install Docker Desktop, **not** just the CLI docker engine

Do we have any kind of reference / documentation / a vagrant issue for this? If vagrant really doesn't work with the docker CLI engine, it is not documented, and there is no issue, then we should open one. But I really doubt it, I wouldn't expect vagrant to interact with any of the desktop components, it just executes some docker commands. (It certainly is possible that you'd need to restart the terminal or reboot after installing docker to update environment variables / user groups / whatever).

> Vagrant does conflict with used ports from other docker containers, even if they are not in use

I don't think vagrant has anything to do with ports here. Do you mean: "If other docker containers on the system are configured to also use port 8000, this causes conflicts"? In that case, I'd expect users to realize this, since they are probably aware of the containers they have on their system. For a fresh docker installation, port 8000 shouldn't be blocked, right?
-> I'd vote to remove this.

Reply:
I do not know why it did not work one year ago, maybe I did something wrong with docker engine. I will remove it. Until someone complains, I cannot test this on a different mac without braking my setup.

If you have conainer a with host port 8000 and b with 8000, docker will allow that they exist simultaneously, with only running one at the same time. Vagrant does not like that, I thought it would be good to note that down because we have a rather unconventional docker setup.

Reply:
The actual containers running in the end shouldn't be special in any way. I wouldn't expect that vagrant has anything to do with ports other than forwarding this information to docker.

With vagrant, the container will run `tail -f /dev/null` indefinitely, so its running (and occupying) the port at any point between `vagrant up` and `vagrant halt` -- if you do not run `vagrant halt`, it will remain occupied. Is it possible that this was the cause of your problem?

Reply:
Create a random docker container using a port of your choice via EXPORT or docker compose. Shut down the container but do not remove it. Use a secondary evap install. Try to use vagrant with docker like the tutorial says on the same port, which now should be usable. Vagrant does not like that.

Reply:
I've opened https://github.com/hashicorp/vagrant/issues/13110, let's see what they say. In any case, I think the error is clear enough that (power) users should understand what's going on.